src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -343,12 +329,12 @@ std::vector<ov::SoPtr<ov::ITensor>> SyncInferRequest::get_tensors(const ov::Outp
 }
 
 const ov::Output<const ov::Node>& SyncInferRequest::get_internal_port(const ov::Output<const ov::Node>& port) const {
-    auto name = get_port_name(port);
+    auto index = find_port(port).idx;
     bool is_input = ov::op::util::is_parameter(port.get_node());
```

Comment:
we can reuse `is_input()` from `find_port(port)` instead of having extra `ov::op::util::is_parameter(port.get_node())`